tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/asynchronous/AsyncClassLevelClient.java

```diff
@@ -63,7 +63,7 @@
 
         Throwable exception = null;
         try {
-            waitCondition.get(1000, TimeUnit.MILLISECONDS);
+            waitCondition.get(1000, TimeUnit.SECONDS);
```

Comment:
@carlosdlr can you add a comment to say "This timeout will normally be ignored as the tests will time out before this timeout. The reason for this large value is to ensure test timeout kicks in before this one."?

Reply:
Done.